api/program/programRouter.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const DB = require('../utils/db-helper');
+const router = express.Router();
+const { requireAdmin } = require('../middleware/authorization');
+
+router.get('/', (req, res) => {
+  DB.findAll('programs')
+    .then((programs) => {
+      res.status(200).json(programs);
+    })
+    .catch((err) => {
+      res.status(500).json({ error: err.message });
+    });
+});
+
+router.get('/:id', (req, res) => {
+  DB.findById('programs', req.params.id)
+    .then((programs) => {
```

Comment:
We're returning a 200 ok when the request is for a program id that doesn't exist, but we should return a 404 not found. Since Postgres won't error out on null results, we just need to do a check here e.g. 
`if (programs) { return 200 } else { return 404 }` 
Can see the profiles router for an example.